instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -207,16 +207,16 @@ public Duration getDuration(String name, Duration defaultValue) {
    * name} has not been configured. The format of the original value must be comma-separated, e.g.
    * {@code one,two,three}.
    */
-  public List<String> getListProperty(String name) {
-    return getListProperty(name, Collections.emptyList());
+  public List<String> getList(String name) {
+    return getList(name, Collections.emptyList());
   }
 
   /**
    * Returns a list-valued configuration property or {@code defaultValue} if a property with name
    * {@code name} has not been configured. The format of the original value must be comma-separated,
    * e.g. {@code one,two,three}.
    */
-  public List<String> getListProperty(String name, List<String> defaultValue) {
+  public List<String> getList(String name, List<String> defaultValue) {
```

Comment:
I used simpler `getList`/`getMap` names instead of `getCommaSeparatedValues`/`getCommaSeparatedMap` that SDK uses -- all other method names are consistent with the `ConfigProperties` interface.

Reply:
Seems like a nice change we can make to the SDK too (since it's alpha)